### ISSUE TITLE
Some fonts in PI are too small

### ISSUE THREAD

**Issue by cinnamon-msft** [microsoft/devhome#3159]
### Steps to reproduce the bug

Launch PI as a bar and look at the expanded view

### Expected result

Font size should be at least 12pt or 1em in order to be accessible

### Actual result

App title, percentages, and clipboard labels are too small:
![image](https://github.com/microsoft/devhome/assets/48369326/3ac8bd0f-6fc4-4618-836a-0e7c95676864)
![image](https://github.com/microsoft/devhome/assets/48369326/cae2d5e2-cd57-4369-83e6-5cecd3e8a43e)
![image](https://github.com/microsoft/devhome/assets/48369326/238e051c-af16-41be-b7e6-517e94cfcbad)
![image](https://github.com/microsoft/devhome/assets/48369326/d46f0972-a82c-446b-8b30-c63b33d78ebb)
